spec/liquid_tags/link_tag_spec.rb

```diff
@@ -19,20 +19,21 @@ def generate_new_liquid(slug)
 
   def correct_link_html(article)
     tags = article.tag_list.map { |t| "<span class='ltag__link__tag'>##{t}</span>" }.reverse.join
-    <<-HTML
-      <div class='ltag__link'>
-        <a href='#{article.user.path}' class='ltag__link__link'>
-          <div class='ltag__link__pic'>
-            <img src='#{ProfileImage.new(article.user).get(150)}' alt='#{article.user.username} image'/>
-          </div></a>
-          <a href='#{article.path}' class='ltag__link__link'>
-            <div class='ltag__link__content'>
-              <h2>#{ActionController::Base.helpers.strip_tags(article.title)}</h2>
-              <h3>#{article.user.name}</h3>
-              <div class='ltag__link__taglist'>#{tags}</div>
-            </div>
-        </a>
-      </div>
+    <<~HTML
```

Comment:
nit: Something happened to the spacing here?

Reply:
fixed spacing in recent commit